code/game/objects/items/devices/scanners/sequence_scanner.dm

```diff
@@ -15,65 +15,78 @@
 	throw_speed = 3
 	throw_range = 7
 	custom_materials = list(/datum/material/iron=200)
+
 	var/list/discovered = list() //hit a dna console to update the scanners database
 	var/list/buffer
 	var/ready = TRUE
 	var/cooldown = 200
 
-/obj/item/sequence_scanner/attack(mob/living/M, mob/living/carbon/human/user)
+/obj/item/sequence_scanner/attack(mob/living/mob, mob/living/carbon/human/user)
```

Comment:
it'd be preferable if another var name was chosen instead of `mob`, like `scanned` or something